includes/bootstrap.php

```diff
@@ -216,6 +217,77 @@ function() {
 	1
 );
 
+
+/**
+ * Add a deactivation modal when deactivating the plugin.
+ *
+ * @since x.x.x
+ */
+add_action(
```

Comment:
This will need to use a named function rather than a closure. That will allow developers to unhook it from the site if they wish.

Reply:
Fixed: https://github.com/10up/distributor/pull/540/commits/38c8d1565f44c3e909aafcba712d1172eabb8c3c